Xamarin.Essentials/Xamarin.Essentials.csproj

```diff
@@ -1,7 +1,8 @@
 ﻿<Project Sdk="MSBuild.Sdk.Extras/2.0.29">
   <PropertyGroup>
-    <TargetFrameworks Condition=" '$(OS)' == 'Windows_NT' ">netstandard1.0;netstandard2.0;Xamarin.iOS10;MonoAndroid60;MonoAndroid70;MonoAndroid71;MonoAndroid80;MonoAndroid81;MonoAndroid90;uap10.0.16299</TargetFrameworks>
-    <TargetFrameworks Condition=" '$(OS)' != 'Windows_NT' ">netstandard1.0;netstandard2.0;Xamarin.iOS10;MonoAndroid60;MonoAndroid70;MonoAndroid71;MonoAndroid80;MonoAndroid81;MonoAndroid90;</TargetFrameworks>
+    <TargetFrameworks></TargetFrameworks>
+    <TargetFrameworks Condition=" '$(OS)' == 'Windows_NT' ">netstandard1.0;netstandard2.0;Xamarin.iOS10;MonoAndroid60;MonoAndroid70;MonoAndroid71;MonoAndroid80;MonoAndroid81;MonoAndroid90;uap10.0.16299;tizen40;</TargetFrameworks>
```

Comment:
@jamesmontemagno I like that this is clean, but by doing this we require everyone who builds essentials to have Tizen SDK installed.  Do we want to do this? or should we add a separate project (and therefore separate nupkg :/ )

Reply:
Do not need to install Tizen SDK to build tizen project.

Tizen TFM is already integrated into the Nuget.Client.

 Tizen SDK installation is only required if you want to use the emulator or device manager, and so on.